test/annexB/built-ins/String/prototype/trimLeft/length.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2017 Valerie Young. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: pending
```

Comment:
I think it's safe to use `sec-string.prototype.trimleft` for the esid. This applies to the other files too